README.md

```diff
@@ -259,6 +259,7 @@ x-transportd:
       - 509
       - 510
       - 511
+  retryafter: # honor the 429 Retry-After header, when present and parsable
```

Comment:
task: Please add a datatype here, like the other plugins.

Reply:
OOps, this didn't really belong here anyway.  This feature is either present or not, but has no config options.